sdk/core/Azure.Core/tests/TestFramework/JsonStream.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace Azure.Core.Testing
+{
+    public class JsonStream : IDisposable
```

Comment:
Are we sure we need this type in shared code?

Reply:
I had planned to beef it up eventually, but can move it out for now if you'd like. Figured it'd be a good helper class.